### DISCUSSION TITLE
"Pokeball to use for catching" could use more options

### DISCUSSION THREAD

**Discussion by Mystenigma** [Farigh/pokeclicker-automation#299] [Ideas]
As the title says, I feel like only having three options is kinda limiting, it would be nice to be able to select more specialised balls if we so choose, then a generic ball to fall back on if that runs out. Of course, it'd also be nice then to be able to automate acquiring these balls too, but for now it'd just be nice to actually use the hundreds of repeat balls I've ended up with for the Pokerus focus.  It could also be useful in niche situations to be able to turn off the automatic ball selection altogether so it only uses your existing Pokeball selections.

**Comment by Farigh** [comment]
Other balls are not purchasable using pokedollars, so it would be a bit trickier, this is part of the issue #286
